python/cog/server/http.py

```diff
@@ -195,9 +197,6 @@ async def predict_idempotent(
                     )
                 ]
             )
-
-        # We've already checked that the IDs match, now ensure that an ID is
-        # set on the prediction object
```

Comment:
Any particular reason you've removed this comment?

Reply:
Oh I see it got removed when commit 1 added something, and then commit 2 removed it again but took this comment with it. Maybe rebase all of those changes out?